### ISSUE TITLE
When score contains pickup followed by multimeasure rest, osmd.cursor.next() produces error 

### ISSUE THREAD

**Issue by michaelcchu** [opensheetmusicdisplay/opensheetmusicdisplay#1327]
Steps to reproduce:

1. Open [demo](https://opensheetmusicdisplay.github.io/demo/).
2. Upload this test file: [test_pickup_multimeasure_rest.zip](https://github.com/opensheetmusicdisplay/opensheetmusicdisplay/files/10722778/test_pickup_multimeasure_rest.zip)

<img height=80 src="https://user-images.githubusercontent.com/50280000/218476122-f97202df-c2f3-4a06-8a48-1b375a9e82a9.png">

3. Click "Show", then click "Next", then click "Next".
4. Error appears in console:

<img height=80 src="https://user-images.githubusercontent.com/50280000/218474974-2753fb36-d2b9-468c-8a10-3c192e2b4254.png">

**Comment by sschmidTU** [comment]
This doesn't happen in the local demo, despite having basically no changes currently, not sure why. Hard to test unfortunately.

**Comment by michaelcchu** [comment]
Thanks for your feedback! I downloaded the most recent version of osmd and ran osmd locally, and the error goes away. I think I hadn't downloaded osmd since January 28th. So now I should have the most recent version: 1.7.4.

Out of curiosity, does the error occur for you on the website? (If you open the URL [https://opensheetmusicdisplay.github.io/demo/](https://opensheetmusicdisplay.github.io/demo/) in Google Chrome and test there, does the error occur?)

What version of osmd is the website using?

**Comment by sschmidTU** [comment]
Yes, it does. The demo is 1.7.4, and as I said, there are almost no changes in the local version since then, so I'm not sure why it only happens in the demo.

**Comment by sschmidTU** [comment]
Oh, I think this was fixed in 9242aafac1c17602fc163abaf1968e7c01b64631, didn't expect that.
So, this should be fixed in the next release.

**Comment by sschmidTU** [comment]
There was actually a bug in the multiple rest measure detection that ignored when the first measure in the sheet was a pickup measure.

This is now fixed, and now it also works without the null check in 9242aafac1c17602fc163abaf1968e7c01b64631.

Also, instead of both cursor positions showing this:
<img width="239" alt="image" src="https://user-images.githubusercontent.com/33069673/219248998-b1429ea5-9869-4eaf-bbe7-9e8abb05d773.png">

It now shows two different positions (for the two measures):
<img width="240" alt="image" src="https://user-images.githubusercontent.com/33069673/219249082-18b6147e-13f5-4dad-998b-eaf823e8a443.png">

<img width="238" alt="image" src="https://user-images.githubusercontent.com/33069673/219249098-c8d28167-429f-4df0-aa1b-71ec36327c21.png">

**Comment by sschmidTU** [comment]
There was actually a bug in the multiple rest measure detection that ignored when the sheet contained pickup/implicit measures, which caused the cursor issue.

This is now fixed, and now it also works without the null check in 9242aafac1c17602fc163abaf1968e7c01b64631.

Also, instead of both cursor positions showing this:
<img width="239" alt="image" src="https://user-images.githubusercontent.com/33069673/219248998-b1429ea5-9869-4eaf-bbe7-9e8abb05d773.png">

It now shows two different positions (for the two measures):
<img width="240" alt="image" src="https://user-images.githubusercontent.com/33069673/219249082-18b6147e-13f5-4dad-998b-eaf823e8a443.png">

<img width="238" alt="image" src="https://user-images.githubusercontent.com/33069673/219249098-c8d28167-429f-4df0-aa1b-71ec36327c21.png">
